client-sdk/ts-web/core/src/hdkey.ts

```diff
@@ -0,0 +1,89 @@
+import {sha512} from 'js-sha512';
```

Comment:
Perhaps it would make sense to slip SLIP-0010 implementation into a separate package?
Or is there an existing JS implementation that would be suitable?

Reply:
> I'm pretty happy with the exposed API for HDKey - the underlying implementation of SLIP-0010 can be changed in the future if we want to extract my implementation while being a non-breaking change and without impacting the exposed API.
> 

Ok, let's leave it as is for now.

> The others I found were already part of specific crypto's ecosystems, in JS without typings, had no tests, or a combination of those factors. (And I've seen none using dataviews & typed-arrays exclusively instead of buffer)

Cool, thanks for the explanation.